src/linear_algebra/pi_tensor_product.lean

```diff
@@ -306,6 +306,18 @@ variables {R} {s}
 
 @[simp] lemma mk_apply (f : Π i, s i) : mk R s f = tprod R f := rfl
 
+@[ext]
+theorem ext {φ₁ φ₂ : (⨂[R] i, s i) →ₗ[R] E} (H : ∀ f, φ₁ (tprod R f) = φ₂ (tprod R f)) : φ₁ = φ₂ :=
```

Comment:
Based on #5200, this should be stated as:
```suggestion
theorem ext {φ₁ φ₂ : (⨂[R] i, s i) →ₗ[R] E} (H : ∀ f, φ₁.comp_multilinear_map (mk R s) = φ₂.comp_multilinear_map (mk R s) : φ₁ = φ₂ :=
```
The version you have here is obtained by chaining that with `multilinear_map.ext`, which the `ext` tactic does automatically.

Reply:
The chaining doesn't happen automatically: if I prove that one, then the proof of `lift.unique` breaks. Given that, I think it's better to keep the original version, it's a lot closer to what one would expect from an `ext` lemma.

Reply:
Huh, that's odd - I'll try this out locally. If it doesn't work automatically, there's another ext lemma missing somewhere.

Reply:
Done in #5419